contracts/oracles/TwapOracle.sol

```diff
@@ -161,8 +182,16 @@ contract TwapOracle is OwnableUpgradeable, TwapInterface {
      * @custom:error Missing error is thrown if token config does not exist
      */
     function updateTwap(address vToken) public returns (uint256) {
-        // VBNB token doesn't have `underlying` method, vBNB's underlying token is wBNB
-        address asset = address(vToken) == vBnb ? WBNB : VBep20Interface(vToken).underlying();
+        address asset;
+
+        if (address(vToken) == vBnb) {
+            asset = WBNB;
+        } else if (address(vToken) == vai) {
+            asset = vai;
+        } else {
+            asset = VBep20Interface(vToken).underlying();
+        }
```

Comment:
We could define an internal function to encapsulate this if/else statement

Reply:
Fixed